### ISSUE TITLE
Unsafe HTML in /content/ is still allowed despite `markup.goldmark.renderer.unsafe: false`

### ISSUE THREAD

**Issue by michalrus** [gohugoio/hugo#6869]
I have this in my `config.yaml`:
```yaml
markup:
  defaultMarkdownHandler: goldmark
  goldmark:
    renderer:
      unsafe: false
```

And in `_index.md`:

```html
<script>alert("💩");</script>
```

And, unfortunately, the alert is being displayed.

Yet:
* [goldmark’s README claims](https://github.com/yuin/goldmark/#html-renderer-options):
     > `html.WithUnsafe` — By default, goldmark **does not render raw HTML** or potentially  dangerous links. With this option, goldmark renders such content as  written.
* there’s this example from @bep suggesting this should not be happening: https://github.com/gohugoio/hugo/issues/6581#issuecomment-562124555

So my question is:
1. How can I turn all HTML off in all Markdown content files?
2. Is there a way in Hugo to disallow `*.html` content files, allowing only `*.md` in the `content/` directory, or should I rather create some guard Bash scripts?

### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.58.0/extended linux/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?
Probably…

**Comment by pamubay** [comment]
Because Goldmark is available since v0.60 release.
Check [Release v0.60.0](https://github.com/gohugoio/hugo/releases/tag/v0.60.0)

> Probably…

Im on 
```
Hugo Static Site Generator v0.65.0-DEV-6A74CBE9/extended linux/amd64 
BuildDate: 2020-02-04T19:00:58+0700
```

Works as expected

**Comment by michalrus** [comment]
Oops, thank you, and sorry, silly me!
